### PR TITLE
develop-Home-tab

### PR DESCRIPTION
**Pull request by erikfTW**
Hi all, here is the draft home tab section - Welcome to Taipy. Please review and let me know if you have any comments. Thank you.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -3,87 +3,87 @@ hide:
...
-- [**"User Manual"**](manuals/about.md) describes the main concepts for Taipy GUI, Taipy Core and Taipy REST. It also  provides information on the deployment of your Taipy app.
-- [**"Reference Manual"**](manuals/reference) documents all the Taipy APIs.
+For data scientists and developers alike, Taipy acts as a catalyst for successful Python endeavors. Whether your aim is a straightforward pilot or a comprehensive application within IDEs or Notebooks, Taipy equips you with all indispensable functionalities. Its architecture is meticulously crafted to truncate both development and deployment timelines significantly.
 
-Taipy requires *Python 3.8* or newer.
+Taipy's web application builder is a robust tool that empowers individuals and enterprises to develop web applications without extensive coding expertise. A primary advantage of Taipy lies in its ability to expedite development across both front-end and back-end domains, encompassing everything from initial prototypes to fully scalable, production-ready applications.
```
Comment (FabienLelaquais) [inline docs/index.md]: "Taipy's web application builder is a robust tool" -> "Taipy is a complete tool"
"web applications" -> "applications"

File: docs/index.md [hunk 2]
```diff
@@ -3,87 +3,87 @@ hide:
...
 
-## Quick Access
+**Taipy Front-End Functionalities:**
```
Comment (FabienLelaquais) [inline docs/index.md]: Remove 'Taipy'

File: docs/index.md [hunk 3]
```diff
@@ -3,87 +3,87 @@ hide:
...
 
-These topics are the most visited ones, we thought you’d like to have a glance at them!
+- Crafting a user interface is accomplished with a foundational understanding of Python programming.
```
Comment (FabienLelaquais) [inline docs/index.md]: I don't understand what you mean

File: docs/index.md [hunk 4]
```diff
@@ -3,87 +3,87 @@ hide:
...
+- Taipy is purposefully engineered to prioritize user-friendliness, resulting in a straightforward and intuitive process of user interface creation.
+- No prior knowledge of web design is necessary, and it eradicates the need for any prerequisites related to CSS and HTML.
+- Leveraging augmented markdown syntax, Taipy aids users in seamlessly generating their desired web pages.
```
Comment (FabienLelaquais) [inline docs/index.md]: Capitalize Markdown please
Taipy -> Taipy GUI

File: docs/index.md [hunk 5]
```diff
@@ -3,87 +3,87 @@ hide:
...
-        <img class="tp-content-card-icon icon-light" src="images/icons/flag-w.svg">
-        <img class="tp-content-card-icon icon-dark" src="images/icons/flag.svg">
-        <h3>Getting Started</h3>
-      </header>
-      <p>
-        This tour shows you how to create an entire application using the two main components of Taipy.
-      </p>
-      <span class="tp-content-card-readmore">Read more</span>
-    </a>
+**Taipy Back-End Functionalities:**
```
Comment (FabienLelaquais) [inline docs/index.md]: Remove Taipy

File: docs/index.md [hunk 6]
```diff
@@ -3,87 +3,87 @@ hide:
...
+
+- Taipy provides extensive support for establishing resilient pipelines capable of managing diverse scenarios.
+- The process of modeling Directed Acyclic Graphs (DAGs) is simplified through Taipy's functionality.
+- Enhancing the overall performance of Taipy applications, the data caching feature is integrated.
+- Facilitating a registry of pipeline executions.
+- Enabling pipeline versioning.
+- Taipy equips users with the ability to monitor and assess the performance of their applications using the KPI tracking tool.
+- Furthermore, Taipy offers a built-in visualization feature for pipelines and their associated data.
+
+## Overview of the Taipy Flow
```
Comment (FabienLelaquais) [inline docs/index.md]: "Overview of using Taipy"? (not too happy with that)

File: docs/index.md [hunk 7]
```diff
@@ -3,87 +3,87 @@ hide:
...
-      <p>
-        This tour shows you how to create an entire application using the two main components of Taipy.
-      </p>
-      <span class="tp-content-card-readmore">Read more</span>
-    </a>
+**Taipy Back-End Functionalities:**
+
+- Taipy provides extensive support for establishing resilient pipelines capable of managing diverse scenarios.
+- The process of modeling Directed Acyclic Graphs (DAGs) is simplified through Taipy's functionality.
+- Enhancing the overall performance of Taipy applications, the data caching feature is integrated.
```
Comment (FabienLelaquais) [inline docs/index.md:29]: "Taipy" -> "data flow"(?)

File: docs/index.md [hunk 8]
```diff
@@ -3,87 +3,87 @@ hide:
...
-    <a class="tp-content-card" href="manuals/about/">
-      <header class="tp-content-card-header">
-        <img class="tp-content-card-icon icon-light" src="images/icons/menu_book-w.svg">
-        <img class="tp-content-card-icon icon-dark" src="images/icons/menu_book.svg">
-        <h3>User Manual</h3>
-      </header>
+
+## Discover how Taipy functions
+
+Taipy consists of two independent components: **Taipy Core** and **Taipy GUI**. While each can be used independently, they synergize remarkably when combined.
```
Comment (FabienLelaquais) [inline docs/index.md]: "Taipy consists of two independent components" -> "Taipy holds two independent Python libraries"

File: docs/index.md [hunk 9]
```diff
@@ -3,87 +3,87 @@ hide:
...
+
+Taipy consists of two independent components: **Taipy Core** and **Taipy GUI**. While each can be used independently, they synergize remarkably when combined. 
+
+<div class="tp-row" style="margin-top: 2rem; margin-bottom: 2rem">
+  <div class="tp-col-12 tp-col-md">
+    <article class="tp-content-card">
+      <h3>Taipy GUI</h3>
       <p>
-        Put Taipy in gear, guided by examples.
+       A user-friendly <strong>Graphical User Interface</strong> for crafting interactive interfaces with Python—no CSS or HTML expertise required.
```
Comment (FabienLelaquais) [inline docs/index.md]: "A user-friendly <strong>Graphical User Interface</strong>" -> "A library"

File: docs/index.md [hunk 10]
```diff
@@ -257,11 +191,13 @@ Below, you'll find a succinct elucidation of each term, contextualized within th
...
-      </div>
+    <a class="tp-pill" href="manuals/cli/">
+      <span>Taipy command-line interface (CLI)</span>
+    </a>
+  </li>
+  <li>
+    <a class="tp-pill" href="manuals/rest/">
+      <span>Expose Taipy's REST APIs</span>
     </a>
   </li>
```
Comment (jrobinAV) [inline docs/index.md:202]:
```suggestion
  </li>
    <li>
    <a class="tp-pill" href="manuals/about/">
      <span>…</span>
      <div class="tp-tooltip">
        <p>Browse the complete list of features.</p>
      </div>
    </a>
  </li>
```

**Review by jrobinAV — APPROVED** [review]

**Review by FabienLelaquais — APPROVED** [review]
We want this integrated now. Probably we'll add a few tweaks, but that's fine!
Thx

File: docs/index.md (hunk 3 above)
Comment (erikfTW) [reply docs/index.md]: I have rewritten it: Creating a user interface requires a solid grasp of Python programming fundamentals.

File: docs/index.md (hunk 6 above)
Comment (erikfTW) [reply docs/index.md]: I see "Overview of the Taipy Flow", which is used in general for topics where you display Schemas / Diagrams. Not sure at the moment what else we could add there?

File: docs/index.md (hunk 7 above)
Comment (erikfTW) [reply docs/index.md:29]: Here I'm not sure to what are you referring to :)

File: docs/index.md (hunk 8 above)
Comment (erikfTW) [reply docs/index.md]: This content is not present anymore in the file.

File: docs/index.md (hunk 9 above)
Comment (erikfTW) [reply docs/index.md]: This content is not present anymore in the file.

File: docs/index.md (hunk 7 above)
Comment (trgiangdo) [reply docs/index.md:29]:
```suggestion
- Enhancing the overall performance of data flow applications, the data caching feature is integrated.
```

File: docs/index.md (hunk 6 above)
Comment (FabienLelaquais) [reply docs/index.md]: I'm not proposing to add anything.
But I have a problem with the generic term "flow".
As if there was a natural flow to use Taipy. 
This might be the case for Core or REST (where you design the configs then create the app that deals with entities...), but certainly not with GUI, or any other piece of the puzzle...
That was my only point.

File: docs/index.md (hunk 7 above)
Comment (FabienLelaquais) [reply docs/index.md:29]: I suspect I'm referring to the "through Taipy's functionality" fragment.
You are saying that Taipy provides a functionality to model DAGs.
I would rephrase this removing the too generic "Taipy" and focus on data flow
